elections/2021/TOC/README.md

```diff
@@ -0,0 +1,121 @@
+---
+title: "Knative TOC Election - 2021 Voters Guide"
+linkTitle: "2021 Voters Guide"
+weight: 30
+type: "docs"
+---
+
+# 2021 Voters Guide - Knative TOC Election
+
+## Purpose
+
+The role of this election is to fill out the one (1) seats due for election this year on the [Knative Technical Oversight Committee]. The elected
+member will serve a two (2) year term.
+
+## Background
+
+This election will shape the future of Knative as a community and project.
+While WGs help shape the individual direction of the project, the
+[TOC Charter] covers the technical direction, health of the project
+and community as a whole.
+
+## Eligibility
+
+Please refer to the [TOC Election Charter] for:
+
+- [Eligibility for candidacy]
+- [Eligibility for voting] 
+
+Eligible Voters list coming soon.
+
+### Schedule
+
+| Date         | Event                    |
+| ------------ | ------------------------ |
+| April 13     | Announcement of Election |
+| April 23     | All candidate nominations and voting exception forms due by 0000 UTC (5pm Pacific) |
+| April 27     | Election Begins via email ballots |
+| May 11       | Election Closes by 0000 UTC (5pm Pacific) |
```

Comment:
I think this Timeline makes sense, we have a release on the 18th, 11th is the pkg cut, but I think this is fine.